recipes/google-cloud-recommendations-ai/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "google-cloud-recommendations-ai" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/google-cloud-recommendations-ai-{{ version }}.tar.gz
+  sha256: 063acb4034469b16c430c77cdb2f49225272d86e82ccbaf9fcabfe77fd0e7a69
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - google-api-core >=1.22.2,<2.0.0dev
+    - proto-plus >=1.15.0
+    # temporary constraint because of incorrect dependencies in
+    # https://github.com/conda-forge/google-api-core-feedstock/pull/74
+    - google-auth >=1.25.0,2.<2.0dev
```

Comment:
I realize I would ideally have `google-api-core` 1.3.2 marked as broken and fix the dependencies, but that feedstock has moved on and I don't want to deal with that headache if it can be avoided.